doc/examples/jupyter_notebook/jupyter.json

```diff
@@ -4,7 +4,7 @@
     "external_port": 30888
   },
   "transform": {
-    "image": "pachyderm_jupyter",
+    "image": "dwhitena/pachyderm_jupyter",
```

Comment:
Yes, I agree.  I mostly changed this to get it working. But I'm going to upload this to our Docker hub and change it.